### ISSUE TITLE
9.1.3.1h/3.1 Modal dialog "File settings" - The input field for the WebDAV URL does not have a label associated with it programmatically. It is recommended to implement a visual label (e.g., "WebDAV Url") using the "label" element, which would be linked to the field using the "for" and "id" attributes.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42212]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-fa5c8d46f9-v3-n1

9.1.3.1h/3.1 Modal dialog "File settings" - The input field for the WebDAV URL does not have a label associated with it programmatically. It is recommended to implement a visual label (e.g., "WebDAV Url") using the "label" element, which would be linked to the field using the "for" and "id" attributes.

**Comment by JuliaKirschenheuter** [comment]
duplicate of https://github.com/nextcloud/server/issues/41835
